src/con_duct/__main__.py

```diff
@@ -43,6 +44,35 @@
     "Reports Written: {num_reports}\n"
 )
 
+ABOUT_DUCT = """
+duct is a lightweight wrapper that collects execution data for an arbitrary
+command.  Execution data includes execution time, system information, and
+resource usage statistics of the command and all its child processes. It is
+intended to simplify the problem of recording the resources necessary to
+execute a command to improve reproducibility, particularly in an HPC environment.
+
+Resource usage is determined by polling, (of frequency sample_interval).
+During execution, duct produces a Newline Deliminated JSON file with one data
+point recorded for each report, (of frequency report-interval).
```

Comment:
dang, forgot that README is produced from here. See recommended changes above

Reply:
We don't have to include the env vars here, the readme is only updated between the markers, if you prefer to move them out of the helptext

Reply:
oh i see, you meant apply the changes to the source, not the destination. got it-- they are applied now